ksql-functional-tests/src/main/java/io/confluent/ksql/test/model/KsqlVersion.java

```diff
@@ -32,17 +32,19 @@
 
   @EffectivelyImmutable
   private static final Comparator<KsqlVersion> COMPARATOR =
-      Comparator.comparing(KsqlVersion::getVersion);
+      Comparator.comparing(KsqlVersion::getVersion)
+      .thenComparingLong(KsqlVersion::getTimestamp);
 
   private final transient String name;
   private final SemanticVersion version;
+  private final long timestamp;
 
   public static KsqlVersion current() {
     return parse(Version.getVersion());
   }
 
   public static KsqlVersion of(final String name, final SemanticVersion version) {
-    return new KsqlVersion(name, version);
+    return new KsqlVersion(name, version, Long.MAX_VALUE);
```

Comment:
What's the purpose of having this default? This constructor doesn't appear to be used.

Reply:
It's so that the comparison works correctly. I want any version without a timestamp to be higher than a version with a timestamp.